src/ppca.jl

```diff
@@ -149,12 +156,15 @@ function bayespca{T<:AbstractFloat}(S::DenseMatrix{T}, mean::Vector{T}, n::Int;
         C⁻¹ = (eye(d) - W*M⁻¹*W')/σ²
         L = (-n/2)*(log(det(C)) + trace(C⁻¹*S))  # (-n/2)*d*log(2π) omitted
         # println("$i] ΔL: $(abs(L_old - L)), L: $L")
-        if abs(L_old - L) < tol
+        chg = abs(L_old - L)
+        if chg < tol
+            converged = true
             break
         end
         L_old = L
         i += 1
     end
+    converged || throw(ConvergenceException(tot, chg, Float64(tol)))
```

Comment:
Why do you call `Float64` here and not above? BTW, it would be better to do `oftype(chg, tol)` in case `chg` ends up being of a different type than `Float64`.

Reply:
woops... should be on both and will change to use `oftype`.